statsd/buffer_pool.go

```diff
@@ -0,0 +1,30 @@
+package statsd
+
+import (
+	"bytes"
+	"sync"
+)
+
+type bufferPool struct {
+	sync.Pool
```

Comment:
I didn't really think about it, I guess it was because bufferPool was embedding sync.Pool. So it would always be called by pointer anyways, plus it makes it easier to instantiate.